migen/test/test_vcd.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+import os
+
+from migen.sim.vcd import VCDWriter
+
+
+class VcdWriter(unittest.TestCase):
+
+    def setUp(self):
+        self.filename = self.id() + ".vcd"
```

Comment:
Use mktemp?

Reply:
I thought about using tempfile.TemporaryFile but the problem is that VCDWriter class creates the file by itself and accept filename only. Are you suggesting creating a safe temporary directory and place temp files in there?

Reply:
Yes, that would be fine.